a12.c

```diff
@@ -0,0 +1,20 @@
+
+#include <stdio.h>
+
+int fibonacci(int n) {
+    int t1 = 0, t2 = 1, nextTerm;
+
+    for (int i = 1; i <= n; ++i) {
+        printf("%d, ", t1);
+        nextTerm = t1 + t2;
+        t1 = t2;
+        t2 = nextTerm;
+    }
+    return 0;
+}
+
```

Comment:
Mi6: write another function which checks given number is a fibonacci number & also update main function & pass random numbers to check if they are fibonacci

Reply:
Working on it...

*This is an automated message generated by Mi6*